src/main/java/org/fcrepo/spec/testsuite/versioning/LdpcvHttpDelete.java

```diff
@@ -53,6 +58,25 @@ public void ldpcvThatAdvertisesDeleteShouldRemoveContainerAndMementos(final Stri
                                         "https://fcrepo.github.io/fcrepo-specification/#ldpcv-delete",
                                         ps);
 
+        //create a versioned resource
+        final Response response = createVersionedResource(uri, info);
+        final String versionedResource = getLocation(response);
+        final String timeMap = getTimeMapUri(response).toString();
+
+        final Response optionsResponse = doOptions(timeMap);
+
+        //check that delete is supported
+        if (hasHeaderValueInMultiValueHeader("Allow", "DELETE", optionsResponse)) {
+            //delete the ldpcv
+            final Response deleteResponse = doDelete(timeMap);
+            assertEquals("Timemap DELETE command failed.", deleteResponse.getStatusCode(), 204);
```

Comment:
The `204` assertion already occurs in ``doDelete().

Reply:
++